x-pack/plugins/ml/server/lib/alerts/alerting_service.ts

```diff
@@ -262,6 +286,8 @@ export function alertingServiceProvider(mlClient: MlClient, datafeedsService: Da
         topRecords: v.record_results.top_record_hits.hits.hits.map((h) => {
           return {
             ...h._source,
+            typical: h._source.typical?.map((t: number) => formatters.numberFormatter(t)),
```

Comment:
For population jobs with an `over` field, `actual` and `typical` will not be present in the top level of `_source`, but are present for each of the items in the `causes` array. 

![image](https://user-images.githubusercontent.com/7405507/141167305-3169a2f1-2637-49f0-9a4e-f9b0e1a84dce.png)

 Could you use the same logic that we use in the anomalies table, where we retrieve the `actual` and `typical` values if there is a single cause - look in `ml/server/models/results_service/build_anomaly_table_items.js`. It is probably also a good idea to check `source.function_description` to see if we want to display the actual / typical for the detector function. Can we leave actual / typical out of the alert if there is more than one `cause` or if we don't display actual / typical for the detector function?

Reply:
@peteharverson implemented in https://github.com/elastic/kibana/pull/118006/commits/beb0e734a92f3c62356f8d9d1456218b3cdc94fd

Reply:
Tested behavior for records from population jobs with commit https://github.com/elastic/kibana/pull/118006/commits/af145725d3c1aa4720240d58299665eacfb51579 and LGTM